AltexSoft lab 1/scss/style.scss

```diff
@@ -0,0 +1,726 @@
+@import "variables";
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.header {
```

Comment:
![image](https://user-images.githubusercontent.com/84564899/120069752-1ed48580-c090-11eb-8efa-ef23afcee8d1.png)
fix all red points: indents, font-sizes, collapsed elements, elements that do not wrap